dev/integration_tests/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -65,37 +65,37 @@ class _CupertinoAlertDemoState extends State<CupertinoAlertDemo> {
                         + MediaQuery.of(context).padding,
                     children: <Widget>[
                       CupertinoButton.filled(
-                        child: const Text('Alert'),
                         onPressed: () => _onAlertPress(context),
+                        child: const Text('Alert'),
                       ),
                       const Padding(padding: EdgeInsets.all(8.0)),
                       CupertinoButton.filled(
-                        child: const Text('Alert with Title'),
                         padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
                         onPressed: () => _onAlertWithTitlePress(context),
+                        child: const Text('Alert with Title'),
                       ),
                       const Padding(padding: EdgeInsets.all(8.0)),
                       CupertinoButton.filled(
-                        child: const Text('Alert with Buttons'),
                         padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
                         onPressed: () => _onAlertWithButtonsPress(context),
+                        child: const Text('Alert with Buttons'),
                       ),
                       const Padding(padding: EdgeInsets.all(8.0)),
                       CupertinoButton.filled(
-                        child: const Text('Alert Buttons Only'),
                         padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
                         onPressed: () {
                           showDemoDialog(
                             context: context,
                             child: const CupertinoDessertDialog(),
                           );
                         },
+                        child: const Text('Alert Buttons Only'),
                       ),
                       const Padding(padding: EdgeInsets.all(8.0)),
                       CupertinoButton.filled(
-                        child: const Text('Action Sheet'),
                         padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
                         onPressed: () => _onActionSheetPress(context),
+                        child: const Text('Action Sheet'),
```

Comment:
for these cases either way seems fine. The updated way on the right will look better with UI guides.